### PR TITLE
feat: preference to show/hide available seats count in events

### LINKED ISSUE #11065: Feature to not show "x seats available"
### Is your proposal related to a problem?

- Displaying number of seats when seats feature is enabled handicaps certain markerting technique like FOMO.
- It also gives away the business's internal insight to competitors.

(Write your answer here.)

### Describe the solution you'd like

- Provide a flag/preference to not show the "x seats available" caption/copy. Screenshot below

(Write your answer here.)

### Additional context

(Write your answer here.)

### Requirement/Document

(Share it here.)

image

### PR DESCRIPTION
**Pull request by shubhams167**
## What does this PR do?

Closes #11065

This PR adds a feature to show a checkbox called "Show the number of available seats" which when enabled would show the exact number of available seats for an event otherwise shows placeholder texts like "Seats available", "Seats filling fast", or "Seats almost full".

This checkbox will be *checked* by default for backward compatibility with older events.

Screenshot 2023-09-03 at 2 36 33 PM

## Screenshots:

**Booking an event with less than 50% seat occupancy**

Screenshot 2023-09-03 at 2 17 56 PM
Screenshot 2023-09-03 at 2 18 11 PM

**Booking an event with more than 50% occupancy**

Screenshot 2023-09-03 at 2 21 56 PM
Screenshot 2023-09-03 at 2 22 09 PM

**Booking an event with more than 83% occupancy**

Screenshot 2023-09-03 at 2 27 18 PM
Screenshot 2023-09-03 at 2 27 27 PM

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/features/bookings/components/AvailableTimes.tsx [hunk 1]
```diff
@@ -110,15 +114,22 @@ export const AvailableTimes = ({
...
+                    ? `${availableSeatsCount} ${t("seats_available", {
+                        count: availableSeatsCount,
+                      })}`
+                    : isNearlyFull
+                    ? t("seats_nearly_full")
+                    : isHalfFull
+                    ? t("seats_half_full")
+                    : t("seats_available", {
+                        count: availableSeatsCount,
+                      })}
```
Comment (sean-brydon) [inline packages/features/bookings/components/AvailableTimes.tsx:132]: Do you think we could make this into a component of some sort? Seems to be repeated across the app in a few places

Comment (shubhams167) [reply packages/features/bookings/components/AvailableTimes.tsx]: That's a good idea! I've refactored the code to create a component out of it.

**Comment by PeerRich** [comment]
this is really cool!

**Review by CarinaWolli — APPROVED** [review]
Great new feature! Code looks good, thank you for your contribution 🙏

**Pull request merged**